tests/dragonfly/__init__.py

```diff
@@ -52,6 +52,7 @@ def stop(self, kill=False):
         except subprocess.TimeoutExpired:
             print("Unable to terminate DragonflyDB gracefully, it was killed")
             proc.kill()
+            proc.communicate()
```

Comment:
why not print the results? https://docs.python.org/3/library/subprocess.html#subprocess.Popen.communicate
or does it print it internally?

Reply:
Did you see the print above when test run? and where did you see the port print?
I found this fail for the disconnect replication fail and I dont see those prints
https://github.com/dragonflydb/dragonfly/actions/runs/4335282784/jobs/7569761257#step:6:1582

Reply:
Oh, its unrelated sorry, I just added this line for correctness